src/main/java/datawave/microservice/audit/replay/ReplayClient.java

```diff
@@ -0,0 +1,409 @@
+package datawave.microservice.audit.replay;
```

Comment:
I like this client.  It is relatively clean and simple.  Why is this not part of AuditClient however?  It seems there could be some shared code.

Reply:
> I like this client. It is relatively clean and simple. Why is this not part of AuditClient however? It seems there could be some shared code.

I could possibly see refactoring both *Client classes, and have both derived from a common AbstractClient or something like that. But , since the two clients are exposing different REST services (**audit/v1/** vs. **replay/v1/**), seems like they'd share only a very small amount of code at best. I suppose it could be worth it though, if we foresee a need for more audit-related client services down the road(?)

Reply:
I guess there's no reason it couldn't be rolled in.  I don't think most people who use the audit client will need audit replay functionality though as that's more of an administrative interface.  The endpoints are different too.

Reply:
Also, please don't merge this anytime soon.  I'm still working through testing this out on a running system.